tests/framework/config/client.go

```diff
@@ -33,3 +33,20 @@ type DeleteOptions struct {
 	FromKey bool
 	End     string
 }
+
+type EpStatus struct {
```

Comment:
No need to shorthand
```suggestion
type EndpointStatus struct {
```

Reply:
StatusOptions would like the option of request. But in here, EpStatus is used to represent response of etctctl command  `./etcdctl endpoint status -w json`, just like follow.
./etcdctl endpoint status -w json | jq .  
[
  {
    "Endpoint": "127.0.0.1:2379",
    "Status": {
      "header": {
        "cluster_id": 14841639068965180000,
        "member_id": 10276657743932975000,
        "revision": 1384929520,
        "raft_term": 4
      },
      "version": "3.5.1",
      "dbSize": 37167104,
      "leader": 10276657743932975000,
      "raftIndex": 8,
      "raftTerm": 4,
      "raftAppliedIndex": 8,
      "dbSizeInUse": 37138432
    }
  }
]
`

So I want change `EpStatus` to `EndpointStatus` ranther than `StatusOptions`. WDYT?

Reply:
Ups, based on directory I have assumed that this is option and not struct to store the response. Please move this struct to tests/framework/interface.go

Reply:
Ok. i will move to `tests/framework/interface.go`, and struct name use `EndpointStatus`,

Reply:
> Ups, based on directory I have assumed that this is option and not struct to store the response. Please move this struct to tests/framework/interface.go

 We do not move to `tests/framework/interface.go`, it will  cause `import cycle not allowed`. 
So I create new directory `tests/schema` to save etcdctl's output struct .